packages/collector/test/tracing/opentelemetry/fs-app.js

```diff
@@ -12,14 +12,7 @@ process.on('SIGTERM', () => {
   process.exit(0);
 });
 
-const expect = require('chai').expect;
-
-/**
- * We install the latest version of the collector here locally.
- * This ensures we are using the Opentelemetry production dependencies.
- */
-expect(require.resolve('@instana/collector')).to.contain('opentelemetry/node_modules/@instana/collector');
-require('@instana/collector')();
+require('../../../src')();
```

Comment:
As far as I can see, we now no longer test against our core otel api version, because you now only load the api version from the root or?

What we want to keep IMO is:
- local test app installation of otel api (root) + collector (no root)
- local test app installation of collector only

IMO loading 

> require('../../../src')();

Has no real benefit for a production use case or?